src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -821,6 +970,12 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         return result;
     }
     
+    /**
+     * Returns a list of courseIds for all archived courses for all instructors.
```

Comment:
Just sharing my view, I personally think it makes much more sense to have the method summary in the description rather than in the `@return` tag to quickly grasp what method does both from understandability and structure aspects. But happy to change all, if project thinks otherwise. :)